src/main/kotlin/calculator/Calculator.kt

```diff
@@ -15,10 +14,15 @@ class Calculator(private val expression: String) {
 
         return calculated
     }
+
     private fun calculateNextStep(calculated: Int, separateExpression: Iterator<String>): Int {
         val operation = Operation.operationOf(separateExpression.next())
         val number = separateExpression.next().toInt()
 
         return operation.calc(calculated, number)
     }
+
+    companion object {
+        val EXPRESSION_SEPARATOR = " "
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/158#discussion_r584421941

```suggestion
        private const val EXPRESSION_SEPARATOR = " "
```